social/pusher/package.json

```diff
@@ -22,8 +22,8 @@
         }
     },
     "author": {
-        "name": "Dave Conway-Jones",
-        "email": "ceejay@vnet.ibm.com",
+        "name": "Shaqaruden",
+        "email": "shaqaruden@gmail.com",
```

Comment:
This should be a `contributors` section, not replacing existing author

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors